fastmri_recon/helpers/adversarial_training.py

```diff
@@ -194,3 +207,4 @@ def adversarial_training_loop(
                 epoch_logs[l] = o
         callbacks.on_epoch_end(epoch, epoch_logs)
     callbacks._call_end_hook('train')
+    return pretraining_history
```

Comment:
don't forget the newline at the end of the file (same for elsewhere).
Maybe your editor isn't configured to add it at save, if not try to make it so.